libkbfs/config_local.go

```diff
@@ -875,7 +895,10 @@ func (c *ConfigLocal) Shutdown(ctx context.Context) error {
 	c.BlockServer().Shutdown(ctx)
 	c.Crypto().Shutdown()
 	c.Reporter().Shutdown()
-	err = c.DirtyBlockCache().Shutdown()
+	dirtyBcache := c.DirtyBlockCache()
```

Comment:
I think you need to set `err = nil` here, otherwise the above error might get appended twice